app/controllers/application_controller.rb

```diff
@@ -12,11 +12,11 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActiveRecord::RecordInvalid do |ex|
     flash[:error] =
-      "Fel i formulär:  #{ex.record.errors.full_messages.join '; '}"
+        "Fel i formulär:  #{ex.record.errors.full_messages.join '; '}"
     render referring_action, status: :unprocessable_entity
   end
 
-  rescue_from ActiveRecord::RecordNotFound do
+  rescue_from ActiveRecord::RecordNotFound do |ex|
```

Comment:
Unused block argument - `ex`. You can omit the argument if you don't care about it.